crates/solvers/src/infra/dex/zeroex/dto.rs

```diff
@@ -76,6 +76,8 @@ pub struct Query {
 pub struct Slippage(BigDecimal);
 
 impl Query {
+    // TODO This style makes it really easy to forget something. It'd be better if I
+    // got rid of this and just had a from_domain(config, order, ...)
```

Comment:
This was mostly to keep a `defaults` around with the default query fields. It should probably be split into two types to be more correct.